internal/wrapper/wrapper.go

```diff
@@ -53,6 +53,12 @@ func RunTerraform(args []string, arch string) (int, error) {
 
 	log.Printf("version '%s' matches all constraints", matchingRelease.Version)
 
+	err = checkStateCommand(args, matchingRelease.Version)
+	if err != nil {
```

Comment:
nitpick: since this only has one return value, you can do:

```suggestion
	if err := checkStateCommand(args, matchingRelease.Version); err != nil {
```